<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeControl</class>
 <widget class="QWidget" name="HomeControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="home_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>100</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/rec/img/home.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="light_checkBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator:unchecked {
image: url(:/rec/img/close.png);
}
QCheckBox::indicator:checked {
image: url(:/rec/img/open.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="door_checkBox">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>170</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator:unchecked {
image: url(:/rec/img/close.png);
}
QCheckBox::indicator:checked {
image: url(:/rec/img/open.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="fan_checkBox">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>100</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox::indicator:unchecked {
image: url(:/rec/img/close.png);
}
QCheckBox::indicator:checked {
image: url(:/rec/img/open.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="door_icon_label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/rec/img/door_close.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fan_icon_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/rec/img/fan.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="fan_intensity_text_label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>风扇控制</string>
   </property>
  </widget>
  <widget class="QLabel" name="door_intensity_text_label">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>260</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>门控制</string>
   </property>
  </widget>
  <widget class="QLabel" name="light_intensity_text_label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>260</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(115, 210, 22);</string>
   </property>
   <property name="text">
    <string>灯光控制</string>
   </property>
  </widget>
  <widget class="QLabel" name="light_icon_label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>120</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/rec/img/light_off.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
